### DISCUSSION TITLE
Twilio Integration

### DISCUSSION THREAD

**Discussion by instincttech** [deepgram/community#482] [General help] [answered]
So I been struggling left and right to get deepgram working streaming twilio. All of these projects keep having one issue after another. So I really wanted to get this one working:
https://github.com/deepgram/public-code-examples/tree/master/twilio/node

but everytime i run it I get 
TypeError: Deepgram is not a constructor

anyone shed some insight?

I really can't find one good example using node.js with twilio media streams that just works. Been a battle all day or can somone point to one good decent working example?

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by SandraRodgers** [comment]
Thanks for reaching out! I think you're running into that error because our SDK went through some breaking changes, and this project has not been updated to migrate to the new version of the SDK. I'll take a look at it today and see if I can update it, but in the meantime you can try installing a lower version of the SDK (Try v2.4.0) or update your code to follow this migration guide https://developers.deepgram.com/docs/js-sdk-v2-to-v3-migration-guide

**Comment by SandraRodgers** [comment, marked as answer]
I've updated this code to work now. The node code is up-to-date with the new SDK. I was able to test it and get it working.

https://github.com/deepgram/public-code-examples/tree/master/twilio/node
